pkg/k8s/k8s_serviceaccount.go

```diff
@@ -0,0 +1,49 @@
+package k8s
+
+import (
+	"context"
+	"fmt"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"time"
+)
+
+// CreateServiceAccount creates a service account
+func CreateServiceAccount(name, namespace string, labels map[string]string) error {
+
+	serviceAccount := &v1.ServiceAccount{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: namespace,
+			Labels:    labels,
+		},
+	}
+
+	ctx, cancel := context.WithTimeout(context.Background(), 20*time.Second)
```

Comment:
as you are defining a timeout - you can declare the var a pkg const 
here is the same

Reply:
https://github.com/celestiaorg/knuu/issues/115